Clinical-Genomics/scout
# Improve "update individual" cli command

A user-friendly `update individual` cli command with more update options. fix #2133

The same command in current master allows users to update only path to bam/cram alignments (bam_file key). 
This branch will make it possible to update the following keys:
 - "bam_file",
 - "mt_bam",
 - "vcf2cytosure",
 - "rhocall_bed",
 - "rhocall_wig",
 - "tiddit_coverage_wig",
 - "upd_regions_bed",
 - "upd_sites_bed"


**How to test**:
1. Install branch locally or on hasta stage.
1. Run the command `scout update individual ` and follow the instructions.
1. Try providing non-valid individual name, keys or non-existent path to files and see how the command behaves

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
